### PR TITLE
Add white balance presets for OM System OM-1 camera

### PR DESCRIPTION
**Pull request by sarunasb**
Add OM System to WB presets' JSON schema.
Add WB presets for OM-1 camera.

This overrides earlier PR #11790 for compiled-in presets.

### CODE REVIEW

**Comment by sarunasb** [comment]
It would be great to have this in darktable 4.2. Thank you!

**Comment by TurboGit** [comment]
Can't be in 4.2, just checked and this camera is still not read by rawspeed so there is no point having the WB preset and noise profile merged.

**Comment by kmilos** [comment]
> I don't see how WB and noiseprofiles can be “harmfull” even if rawspeed is lagging.

I have already linked to prior problems (in the related OM-1 noise profile issue) when merging these before maker/model get normalized by rawspeed. Yes, the risk might be low in this case, but still...

Thankfully both WB presets and noise profiles are in JSON files now and easy to add.

**Comment by TurboGit** [comment]
Please never merge master branch into a topic branch. You need to rebase ontop of master. TIA.

**Comment by Nilvus** [comment]
Rawspeed support added. @sarunasb: remains a rebase in a branch and not on master to do on your side. See last comment.

**Comment by Nilvus** [comment]
Sorry, seems that you fix it just before our comments.

**Comment by TurboGit** [comment]
I'll expect this to be done:

```
$ git checkout om-system-wb-presets
$ git rebase master
$ git push -f sarunasb om-system-wb-presets
```

File: .gitmodules [hunk 1]
```diff
@@ -1,7 +1,7 @@
 [submodule "src/external/rawspeed"]
 	path = src/external/rawspeed
-	url = https://github.com/darktable-org/rawspeed.git
-	branch = develop
+	url = https://github.com/sarunasb/rawspeed.git
+	branch = om
```
Comment (kmilos) [inline .gitmodules:4]: The submodule change shouldn't have been committed obviously...

Comment (sarunasb) [reply .gitmodules]: Unfortunately it became obvious enough for me only after the fact. Should now be corrected.

**Comment by phrrk** [comment]
I may be missing something, but I cannot see why the depends:rawspeed label has been re-added (@TurboGit ).  Rawspeed seems to be up to date as it now handles OM-1 raw files.

**Comment by TurboGit** [comment]
> I may be missing something, but I cannot see why the depends:rawspeed label has been re-added (@TurboGit ). Rawspeed seems to be up to date as it now handles OM-1 raw files.

Because it depends on rawspeed, and this will be true for as long as darktable will exist & depends on rawspeed. And anyway it is not yet merged and darktable still cannot handled OM-1.

**Comment by kmilos** [comment]
@TurboGit Good to go now!

**Comment by TurboGit** [comment]
Sure, and we must not forget to merge this in 4.2.1 when rawspeed branch is updated there too.

**Review by TurboGit — APPROVED** [review]
Thanks!

**Comment by kmilos** [comment]
Unfortunately it looks like the backport slipped through the cracks, can you please change the milestone to 4.4 then? @TurboGit

**Comment by TurboGit** [comment]
Done.

**Pull request merged**